app/src/main/java/ch/epfl/sweng/swenggolf/offer/ListAnswerAdapter.java

```diff
@@ -128,38 +138,45 @@ private void setUserData(User value, AnswerViewHolder holder) {
     private void setupFavorite(final AnswerViewHolder holder, int position) {
         ImageButton favButton = holder.getContainer().findViewById(R.id.favorite);
         favButton.setContentDescription("fav" + Integer.toString(position));
```

Comment:
I think this line is useless now since the test don't use it anymore.

Reply:
It seems it's useful because tests don't pass without it.